test/unit/asrouter/ASRouter.test.js

```diff
@@ -1531,13 +1531,18 @@ describe("ASRouter", () => {
     describe(".setupTrailhead", () => {
       let getBoolPrefStub;
       let setStringPrefStub;
+      let setBoolPrefStub;
 
       beforeEach(() => {
-        getBoolPrefStub = sandbox.stub(global.Services.prefs, "getBoolPref").withArgs(TRAILHEAD_CONFIG.DID_SEE_ABOUT_WELCOME_PREF).returns(true);
+        getBoolPrefStub = sandbox.stub(global.Services.prefs, "getBoolPref");
+        getBoolPrefStub.withArgs(TRAILHEAD_CONFIG.DID_SEE_ABOUT_WELCOME_PREF).returns(true);
+        getBoolPrefStub.withArgs(TRAILHEAD_CONFIG.TRIPLETS_ENROLLED_PREF).returns(false);
         setStringPrefStub = sandbox.stub(global.Services.prefs, "setStringPref");
+        setBoolPrefStub = sandbox.stub(global.Services.prefs, "setBoolPref");
       });
 
-      const configWithExperiment = {experiment: "interrupts", interrupt: "join", triplet: "privacy"};
+      const configWithInterruptExpriment = {experiment: "interrupts", interrupt: "join", triplet: "privacy"};
+      const configWithTripletExpriment = {experiment: "triplets", interrupt: "join", triplet: "privacy"};
```

Comment:
nit: naming `configWithInterruptsExperiment` ("s" and "expriment" typo) same for Triplets